runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -654,17 +654,17 @@ impl<Cfg: Config> Module<Cfg> {
         Self::get_storage(ctx, body.address, body.index)
     }
 
-    #[handler(query = "evm.Code")]
+    #[handler(query = "evm.Code", allow_confidential)]
```

Comment:
Code isn't confidential though? So `allow_confidential` only enables access to confidential state via the key manager. Since this query does not access the key manager, this shouldn't be needed.

Reply:
Thanks. May I then suggest renaming `allow_confidential` to `permit_km` or something?